csrc/quantization/compressed_tensors/int8_quant_kernels.cu

```diff
@@ -28,15 +28,34 @@ namespace vllm {
 template <typename scalar_t, typename scale_type>
 __global__ void static_scaled_int8_quant_kernel(
     const scalar_t* __restrict__ input, int8_t* __restrict__ out,
-    scale_type scale, const int hidden_size) {
+    scale_type inverted_scale, const int hidden_size) {
   const int tid = threadIdx.x;
   const int token_idx = blockIdx.x;
+  const float4* vectorized = reinterpret_cast<const float4*>(input);
+  const int traverse_space = hidden_size >> 2;
 
-  for (int i = tid; i < hidden_size; i += blockDim.x) {
-    out[token_idx * hidden_size + i] =
-        float_to_int8_rn(((float)input[token_idx * hidden_size + i]) / scale);
+#pragma unroll 4
```

Comment:
This will improve ILP by reordering the instructions a bit to reduce the impact of the warp stalling from the memory load operation.